--- docs/labelImg.py.original	2025-10-03 16:24:37
+++ ./.venv/lib/python3.11/site-packages/labelImg/labelImg.py	2025-10-03 15:50:13
@@ -962,7 +962,7 @@
     def scroll_request(self, delta, orientation):
         units = - delta / (8 * 15)
         bar = self.scroll_bars[orientation]
-        bar.setValue(bar.value() + bar.singleStep() * units)
+        bar.setValue(int(bar.value() + bar.singleStep() * units))
 
     def set_zoom(self, value):
         self.actions.fitWidth.setChecked(False)
